### ISSUE TITLE
FixMisencodedBaseQualityReads needs tests

### ISSUE THREAD

**Issue by akiezun** [broadinstitute/gatk#1230]
include CRAM tests too

**Comment by droazen** [comment]
Closing as obsolete

**Comment by cmnbroad** [comment]
@droazen Curious why this was closed as obsolete ? We still have the tool, there are no tests, and I don't see an obvious Picard equivalent, at least with a similar name.

**Comment by droazen** [comment]
@cmnbroad Yes, you're correct -- this was closed in error, as I thought it was a Picard tool. Re-openning.

**Comment by cmnbroad** [comment]
When the tests are added, we should consider changing this tool to use the transform support built into the framework instead of manually applying it.
